Description: Change references to the binary from stunnel to stunnel4
Forwarded: not-needed
Author: Julien Lemoine <speedblue@debian.org>,
  Luis Rodrigo Gallardo Cruz <rodrigo@debian.org>
Last-Update: 2014-08-08

--- a/src/stunnel3.in
+++ b/src/stunnel3.in
@@ -22,7 +22,7 @@
 use Getopt::Std;
 
 # Configuration - path to stunnel (version >=4.05)
-$stunnel_bin='@prefix@/bin/stunnel';
+$stunnel_bin='@prefix@/bin/stunnel4';
 
 # stunnel3 script body begins here
 ($read_fd, $write_fd)=POSIX::pipe();
--- a/doc/stunnel.8
+++ b/doc/stunnel.8
@@ -61,8 +61,8 @@
 .\}
 .\" ========================================================================
 .\"
-.IX Title "STUNNEL 8"
-.TH STUNNEL 8 "2014.05.08" "5.02" "stunnel"
+.IX Title "STUNNEL4 8"
+.TH STUNNEL 8 "2014.05.08" "5.02" "stunnel4"
 .\" For nroff, turn off justification.  Always turn off hyphenation; it makes
 .\" way too many mistakes in technical documents.
 .if n .ad l
--- a/tools/stunnel.conf-sample.in
+++ b/tools/stunnel.conf-sample.in
@@ -9,17 +9,17 @@
 
 ; A copy of some devices and system files is needed within the chroot jail
 ; Chroot conflicts with configuration file reload and many other features
-;chroot = @localstatedir@/lib/stunnel/
+;chroot = @localstatedir@/lib/stunnel4/
 ; Chroot jail can be escaped if setuid option is not used
 setuid = nobody
 setgid = @DEFAULT_GROUP@
 
 ; PID file is created inside the chroot jail (if enabled)
-;pid = @localstatedir@/run/stunnel.pid
+;pid = @localstatedir@/run/stunnel4.pid
 
 ; Debugging stuff (may be useful for troubleshooting)
 ;debug = 7
-;output = @localstatedir@/log/stunnel/stunnel.log
+;output = @localstatedir@/log/stunnel4/stunnel.log
 
 ; **************************************************************************
 ; * Service defaults may also be specified in individual service sections  *
--- a/doc/stunnel.pl.8
+++ b/doc/stunnel.pl.8
@@ -61,8 +61,8 @@
 .\}
 .\" ========================================================================
 .\"
-.IX Title "STUNNEL.PL 8"
-.TH STUNNEL.PL 8 "2014.05.08" "5.02" "stunnel"
+.IX Title "STUNNEL4.PL 8"
+.TH STUNNEL.PL 8 "2014.05.08" "5.02" "stunnel4"
 .\" For nroff, turn off justification.  Always turn off hyphenation; it makes
 .\" way too many mistakes in technical documents.
 .if n .ad l
--- a/doc/stunnel.fr.8
+++ b/doc/stunnel.fr.8
@@ -61,8 +61,8 @@
 .\}
 .\" ========================================================================
 .\"
-.IX Title "STUNNEL.FR 8"
-.TH STUNNEL.FR 8 "2014.03.09" "5.01" "stunnel"
+.IX Title "STUNNEL4.FR 8"
+.TH STUNNEL.FR 8 "2014.03.09" "5.01" "stunnel4"
 .\" For nroff, turn off justification.  Always turn off hyphenation; it makes
 .\" way too many mistakes in technical documents.
 .if n .ad l
--- a/tools/stunnel.init.in
+++ b/tools/stunnel.init.in
@@ -1,6 +1,6 @@
 #! /bin/sh -e
 ### BEGIN INIT INFO
-# Provides:          stunnel
+# Provides:          stunnel4
 # Required-Start:    $local_fs $remote_fs
 # Required-Stop:     $local_fs $remote_fs
 # Should-Start:      $syslog
@@ -14,8 +14,8 @@
 #                    to completely disable *all* tunnels.
 ### END INIT INFO
 
-DEFAULTPIDFILE="/var/run/stunnel.pid"
-DAEMON=@prefix@/bin/stunnel
+DEFAULTPIDFILE="/var/run/stunnel4.pid"
+DAEMON=@prefix@/bin/stunnel4
 NAME=stunnel
 DESC="SSL tunnels"
 OPTIONS=""
@@ -42,9 +42,9 @@
 }
 
 startdaemons() {
-  if ! [ -d /var/run/stunnel ]; then
-    rm -rf /var/run/stunnel
-    install -d -o stunnel -g stunnel /var/run/stunnel
+  if ! [ -d /var/run/stunnel4 ]; then
+    rm -rf /var/run/stunnel4
+    install -d -o stunnel4 -g stunnel4 /var/run/stunnel4
   fi
   if [ -n "$RLIMITS" ]; then
     ulimit $RLIMITS
@@ -100,9 +100,9 @@
   OPTIONS="-- $OPTIONS"
 fi
 
-[ -f /etc/default/stunnel ] && . /etc/default/stunnel
+[ -f /etc/default/stunnel4 ] && . /etc/default/stunnel4
 if [ "$ENABLED" = "0" ] ; then
-  echo "$DESC disabled, see /etc/default/stunnel"
+  echo "$DESC disabled, see /etc/default/stunnel4"
   exit 0
 fi
 
